src/DynamoCoreWpf/Views/Search/LibraryView.xaml.cs

```diff
@@ -97,8 +97,12 @@ private void OnMemberMouseEnter(object sender, MouseEventArgs e)
             }
 
             FrameworkElement fromSender = sender as FrameworkElement;
-            libraryToolTipPopup.PlacementTarget = fromSender;
-            libraryToolTipPopup.SetDataContext(fromSender.DataContext);
+            var memberVM = fromSender.DataContext as NodeSearchElementViewModel;
```

Comment:
Since I deleted member template and used common template. I had to remove events from old template as well. I added these events to tree item. And since tree item can have tooltip we need additional check.